ssn/rdf/ontology/extensions/sosa-oms.ttl

```diff
@@ -140,31 +140,27 @@ sosa:Procedure
   ogc-ms:implements <http://www.opengis.net/spec/om/3.0/req/obs-cpt/Procedure> ;
   rdfs:isDefinedBy sosa-common: .
 
-sosa-oms:ObservingProcedure 
+sosa:ObservingProcedure 
   ogc-ms:implements <http://www.opengis.net/spec/om/3.0/req/obs-cpt/ObservingProcedure> ;
-  a owl:Class ;
-  rdfs:label "Observing Procedure"@en ;
-  skos:definition """
-  The description of steps performed in order to determine the value of a Property by an Observer (Sensor).
-  """@en ;
-  skos:example "A workflow, protocol, plan, algorithm, or computational method specifying how to make an observation; the description of the process utilized by an observer. This could be a chemical analysis method, a protocol for measuring an object, but could also be a checklist utilized by a human observer during a biodiversity campaign."@en ;
-  skos:scopeNote "The observing procedure cannot describe a sensor instance, but it can describe the sensor type."@en ;
-  rdfs:isDefinedBy sosa-oms: .
+  skos:definition """[ISO 19156:2023] An ObservingProcedure shall be defined as the description of steps performed in order to determine the value of an observableProperty by an Observer.""" ;
+  skos:example """[ISO 19156:2023] An instance of Procedure is a description of the process utilized by an observer. This could be a chemical analysis method, a protocol for measuring an object, but could also be a checklist utilized by a human observer during a biodiversity campaign. Procedure could further describe the algorithms behind simulators or models used to generate a result from other inputs."""@en ;
```

Comment:
```suggestion
  skos:example """[ISO 19156:2023] An instance of Procedure is a description of the process used by an observer. This could be a chemical analysis method, a protocol for measuring an object, or even a checklist used by a human observer during a biodiversity campaign. Procedure could further describe the algorithms behind simulators or models used to generate a result from other inputs."""@en ;
```